pinot-filesystem/src/main/java/org/apache/pinot/filesystem/PinotFS.java

```diff
@@ -74,8 +74,8 @@ public abstract boolean move(URI srcUri, URI dstUri, boolean overwrite)
       throws IOException;
 
   /**
-   * Same as move except the srcUri is not retained. For example, if x/y/z is copied to a/b/c, x/y/z will be retained
-   * and x/y/z will also be present as a/b/c
+   * Same as move except the srcUri is retained. For example, if a file x/y/z is copied to a/b/c, x/y/z will be retained
```

Comment:
Instead of "Same as move" can you mention each case explicitly and describe the behavior? thanks.